gigamon/assets/logs/gigamon.yaml

```diff
@@ -0,0 +1,101 @@
+id: gigamon
+backend_only: true
+facets:
```

Comment:
Depending on the other thread about what processors you require, you should remove all non `gigamon` facets from this list and have the linter add these for you based on the processors you have. 

There are many here from when you had the previous processors but aren't relevant anymore after you removed them

Reply:
@vishalshah-dd , i have made some respective changes and pushed it. Could you review it and let me know if we can connect to complete this over a single call ?

Reply:
Let me know if you want to get on a call, we could definitely arrange that. 

If the facets are "final", then can you please add a `gigamon.` prefix to each facet and then have remappers to map the attributes into the `gigamon.` prefix? The facet paths need to be unique across all integration pipelines so we ask that custom facets be namespaced with some prefix relating to the integration

Reply:
Thanks @vishalshah-dd . Can we connect if you are available now ?:

https://gigamon.zoom.us/j/93045007134?pwd=cFNMWUEwS2ZSNlI3R1hBQzM4OHpEUT09

Reply:
Please ignore Vishal. It's already late here. Can we connect sometime tomorrow ? Wednesday(29th) - 10AM IST ? If not, please propose your time with your mail id to send out an invite. Thanks in advance

Reply:
@Mrudula-Oruganti-Gigamon We are putting the PR review on hold until we have another call with Gigamon to discuss the integration further.